src/test/java/duke/data/task/TaskTest.java

```diff
@@ -0,0 +1,15 @@
+package duke.data.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskTest {
+    @Test
+    public void markAsDone_success(){
+        Task todo = new ToDo("read book");
+        assertEquals("[T][ ] read book", todo.toString());
```

Comment:
Rather than asserting `todo.toString()`, it might be more appropriate to assert the `todo.isDone()` is `false` as the `toString()` messages can change during development and it might cause tests to fail if you forget to update the tests.